ddtrace/tracer/transport.go

```diff
@@ -146,21 +144,23 @@ func (t *httpTransport) send(p *payload) (body io.ReadCloser, err error) {
 	req.Header.Set(traceCountHeader, strconv.Itoa(p.itemCount()))
 	req.Header.Set("Content-Length", strconv.Itoa(p.size()))
 	req.Header.Set(headerComputedTopLevel, "yes")
-	if t, ok := traceinternal.GetGlobalTracer().(*tracer); ok {
-		if t.config.canComputeStats() {
-			req.Header.Set("Datadog-Client-Computed-Stats", "yes")
-		}
-		droppedTraces := int(atomic.SwapUint32(&t.droppedP0Traces, 0))
-		partialTraces := int(atomic.SwapUint32(&t.partialTraces, 0))
-		droppedSpans := int(atomic.SwapUint32(&t.droppedP0Spans, 0))
-		if stats := t.statsd; stats != nil {
-			stats.Count("datadog.tracer.dropped_p0_traces", int64(droppedTraces),
-				[]string{fmt.Sprintf("partial:%s", strconv.FormatBool(partialTraces > 0))}, 1)
-			stats.Count("datadog.tracer.dropped_p0_spans", int64(droppedSpans), nil, 1)
-		}
-		req.Header.Set("Datadog-Client-Dropped-P0-Traces", strconv.Itoa(droppedTraces))
-		req.Header.Set("Datadog-Client-Dropped-P0-Spans", strconv.Itoa(droppedSpans))
-	}
+	// TODO(kjn v2): Figure out the stats. They probably don't belong in the Tracer interface?
```

Comment:
Instead of coming up with a big trace stats API in the tracer, I'm opening this to comment.
I think counting things like dropped P0, partial traces, submitted traces, etc. might be better done *outside* the tracer, with an interface that everyone can talk to.

Reply:
This could be a separate component that is fed from the tracer?

Reply:
@darccio Yes, exactly.

Reply:
Fixed.